src/styles/main.scss

```diff
@@ -1,3 +1,64 @@
+$dayWidth: 100px;
+$dayInWeek: 7;
+$gap: 1px;
```

Comment:
It`s better to create different files. File with your variables should be named as _variables.scss